.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+sudo: required
+before_install:
+- echo $GCLOUD_CREDENTIALS | openssl enc -d -aes-256-cbc -base64 -A -k $GCLOUD_CREDENTIALS_KEY > fhir-org-cds-services.json
+branches:
+  only:
+    - master
+node_js:
+  - '6.11'
```

Comment:
Can we upgrade our node version? 8.9.1 is the latest long term support version.

Reply:
Addressed in https://github.com/cds-hooks/sandbox-cds-services/pull/2/commits/addfff25e47ddd7867a95943c6ba416c992c62a8